src/main/resources/assets/eln2/lang/en_us.json

```diff
@@ -39,5 +39,7 @@
     "waila.eln2.power": "Power",
     "waila.eln2.mode": "Mode",
 
-    "gui.eln2.save": "Save"
+    "gui.eln2.save": "Save",
+
+    "misc.eln2.acknowledge_analytics": "By default, Electrical Age 2 reports analytics to the mod author(s). If you wish to opt-out, please edit your config file. This message will not show again for 1 month."
```

Comment:
We should probably make this mod-agnostic in the future and refer to is as `Age Series Mods`.. granted we probably would have name conflicts. Ideally... this would be in `libage-minecraft` once it exists.